zammad/values.yaml

```diff
@@ -402,6 +402,12 @@ autoWizard:
   #     ]
   #   }
 
+commonLabels: {}
+  # my-label: "value"
+commonAnnotations: {}
+  # my-annotation: "value"
+podLabels: {}
```

Comment:
imho thats not very flexible as you only can set same labels on all pods.
should be available per pod.
in the end would be same as commonLabels, or not?

Reply:
Should be fixed by a36b0b94cd0ec3a25119f61f54c47ae37360a6c7, can you check please?

Reply:
@monotek the final state should allow for anything an admin can want to have regarding labels and annotations, global, global for pods, and pod specific.
Please let me know if this looks ok.

Reply:
Nice :)